pkg/api/pluginproxy/ds_auth_provider.go

```diff
@@ -88,13 +88,6 @@ func getTokenProvider(ctx context.Context, cfg *setting.Cfg, ds *models.DataSour
 	}
 
 	switch authType {
-	case "azure":
```

Comment:
Let's keep the Azure authentication in plugin proxy for a while. Prometheus datasource will likely to start using it. I'm working on Azure authentication in Prometheus and that datasource hasn't migrated to Plugin SDK.